hooks/class.tx_mksanitizedparameters_hooks_PreprocessTypo3Requests.php

```diff
@@ -63,7 +63,10 @@ public function sanitizeGlobalInputArrays()
      */
     protected function getMksanitizedparametersMainClass()
     {
-        return tx_rnbase::makeInstance('tx_mksanitizedparameters');
+        /** @var tx_mksanitizedparameters $class */
+        $class = tx_rnbase::makeInstance('tx_mksanitizedparameters');
```

Comment:
Why is this needed? This is bad practice in my opinion. Use a variable just for a `@var` statement is not worth it.
Can you check if our package [`phpstan-rnbase`](https://github.com/DMKEBUSINESSGMBH/phpstan-rnbase/) fix this problem?

Reply:
Your package don't fix this problem. I can't think of a better solution for this problem.

Can you add this fix to your package, please?

Reply:
Can you open an issue for this, with a reproduceable testcase?

Reply:
[https://github.com/DMKEBUSINESSGMBH/phpstan-rnbase/issues/2](https://github.com/DMKEBUSINESSGMBH/phpstan-rnbase/issues/2)

DONE